packit/config/package_config.py

```diff
@@ -321,14 +321,15 @@ def get_local_specfile_path(directories: Union[List[str], List[Path]]):
     return None
 
 
-def get_specfile_path_from_repo(project: GitProject):
+def get_specfile_path_from_repo(project: GitProject) -> Optional[str]:
     """
     Get the path of the spec file in the given repo if present.
     :param project: GitProject
-    :return: str path of the spec file
+    :return: str path of the spec file or None
     """
-    try:
-        return project.get_files(filter_regex=".*.spec")[0]
-    except Exception as ex:
-        logger.debug(f"Cannot find the spec file: {ex}")
+    spec_files = project.get_files(filter_regex=r".*.spec")
+    logger.debug(f"Spec file(s) found in {project.full_repo_name}: {spec_files}")
```

Comment:
Maybe I would move this behind the if-block.

Reply:
Ok, I removed it as there is one in the `if` block